sqtom/fitting_2d.py

```diff
@@ -85,15 +85,19 @@ def marginal_calcs_2d(jpd_data, as_dict=True):
     g2s = (ns2 - ns) / ns ** 2
     g2i = (ni2 - ni) / ni ** 2
     g11 = (na @ jpd_data @ nb) / (ns * ni)
+    nrf = np.sum([[((i - j) ** 2) * jpd_data[i, j] for i in range(inta)] for j in range(intb)])
```

Comment:
Worth adding to the docstring that this stands for the noise reduction factor (`Returns`)

Reply:
Yup, good point.